### ISSUE TITLE
webp library installation

### ISSUE THREAD

**Issue by eroux** [buda-base/buda-base#32]
The IIIF installation script should install the webp library. In the current state the server can't be compiled because of that

**Comment by MarcAgate** [comment]
issue addressed in commit d6eec55

What we actually do is 1) download the required imageio file and 2) install it via maven install in the local maven repo. This way, the dependency we have in the pom:

```
<dependency>      
        <groupId>iiif.bdrc.io</groupId>  
        <artifactId>webp-imageio</artifactId>  
        <version>0.1.0</version> 
    </dependency>
```

can be effective. It probably would work if we were deploying the jar directly to maven central. I'll test that configuration later. Let's not close the issue.

**Comment by eroux** [comment]
excellent, thanks!
